### DISCUSSION TITLE
Dedicated Server Setup

### DISCUSSION THREAD

**Discussion by Acsere** [official-antistasi-community/A3-Antistasi#2606] [Q&A] [answered]
Maybe I missed it, or maybe I am that inept at this, but I cannot for the life of me figure out how to get a hosted DS to properly run this 3.0 mod version.  Are there detailed instructions I could follow somewhere for proper setup? 

I have the Mod installed server and client side.  I see my server in the list and am able to log in.  I do not, however, see the Antistasi mission available in any of the maps shown.  I am not sure if I need to point my server.cfg to a certain mission file or what.

Sorry for the newbie question here, but any help would be appreciated.

**Comment by HakonRydland** [comment]
can you upload your server rpt?

**Comment by Acsere** [comment, marked as answer]
Bare with me, I am not familiar with GitHub.  Hopefully this is the correct file you are looking for?
[arma3server_2022-12-31_16-00-58.rpt.txt](https://github.com/official-antistasi-community/A3-Antistasi/files/10328129/arma3server_2022-12-31_16-00-58.rpt.txt)

**Reply by HakonRydland** [reply]
no mods where loaded in the session this rpt is from, are you sure you set up the command line correctly?
also what hosting service are you using?

**Reply by Acsere** [reply]
I am not honestly.  I am guessing that the issue is more my ineptness rather than missing something. lol.  I have not messed with Arma servers in many years and I feel old trying to re-figure this out now.

I am using Streamline Servers, and obtained all mods I wanted through their option to download from Steam Workshop.  My file manager shows all the mods listed as well.  I am sure that I failed to enter or update something somewhere.

![image](https://user-images.githubusercontent.com/121737093/210158035-ac7648f0-1f07-49f3-9985-3146b05fa8f0.png)

**Reply by HakonRydland** [reply]
do you have a configuration menu?

**Reply by HakonRydland** [reply]
ok from a quick google it looks like a general server provider and not something focused on arma so im guessing your starting the server with a command, correct?

**Reply by Acsere** [reply]
Your Command Line comment had me looking elsewhere in the server options and I did find the issue.  There is a Commandlind Manager section that appears to auto create command lines based on what is loaded.  I had the default still selected and not the auto configured one.  After switching, it appears to be working now.   I do appreciate the fast assistance!

![image](https://user-images.githubusercontent.com/121737093/210158089-4a54c6b7-5af7-4849-9227-4b6a656b0f56.png)

**Reply by HakonRydland** [reply]
happy you got it working, have fun 😄
